### PR TITLE
fix(desktop): balance refresh rework for edges cases

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Fix edge cases on Cold start

Cap 2026-03-06 at 16 46 23

### ❓ Context

- **JIRA or GitHub link**: https://ledgerhq.atlassian.net/browse/LIVE-27180

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: apps/ledger-live-desktop/src/mvvm/components/TopBar/hooks/useAccountsSyncStatus.ts [hunk 1]
```diff
@@ -21,29 +21,31 @@ export interface AccountsSyncStatus {
...
-  const listOfErrorAccountNames = [...errorTickersSet].join("/");
-  const areAllAccountsUpToDate = errorTickersSet.size === 0;
+  const listOfErrorAccountNames = [...new Set(errorTickers)].join("/");
+  const areAllAccountsUpToDate = errorTickers.length === 0;
+
+  const errorTickersKey = errorTickers.join("/");
+  useEffect(() => {
+    if (!errorTickersKey) return;
+    for (const currency of errorTickersKey.split("/")) {
+      track("SyncError", { currency });
```
Comment (claudiiafg) [inline apps/ledger-live-desktop/src/mvvm/components/TopBar/hooks/useAccountsSyncStatus.ts]: missing the page here
